tests/baselines/reference/ambientConstLiterals.js

```diff
@@ -74,5 +74,5 @@ declare const c9: {
 declare const c10: number[];
 declare const c11: string;
 declare const c12: number;
-declare const c13: string;
-declare const c14: number;
+declare const c13: "abc" | "def";
+declare const c14: 123 | 456;

```

Comment:
Interestingly I think this case came up during our talks about `isolatedDeclarations`. The code for these two is:

```ts
const c13 = Math.random() > 0.5 ? "abc" : "def";
const c14 = Math.random() > 0.5 ? 123 : 456;
```

@RyanCavanaugh @DanielRosenwasser

Reply:
Oh, I just realized who reported the bug, it is directly related!